docs/source/files/bootstrap-vxlan.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Prepare hypervisors vxlan
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
+  become: true
+  tasks:
+    - name: Open VXLAN port
```

Comment:
IMO, we could check if firewall services is running before opening the port.

Reply:
hmm maybe. I didn't have to do this on a freshly installed Stream node though... That playbook is mostly an example of what ppl can do. To be fair, I'd rather see them get a second NIC on a physical, dedicated switch rather than this vxlan thingy....
Let's keep it simple and see how it goes?